platform/drm/cog-platform-drm.c

```diff
@@ -352,6 +352,22 @@ check_drm(void)
     return supported;
 }
 
+static int32_t
+find_crtc_for_encoder(const drmModeRes *resources, const drmModeEncoder *encoder) {
+    int i;
+
+    for (i = 0; i < resources->count_crtcs; i++) {
```

Comment:
Cog is built in C11 mode, which means we can move the declaration of the variable used for loop control inside the first expression of the `for` statement:

```suggestion
    for (int i = 0; i < resources->count_crtcs; i++) {
```